### DISCUSSION TITLE
Windows: How to remove command line from release build

### DISCUSSION THREAD

**Discussion by rth-8** [bevyengine/bevy#6266] [Q&A] [answered]
Hello.
I made my first small game. I built it in release mode (cargo build --release). It works fine, but when I launch the executable, it also opens windows command line with some initial info:

> ←[2m2022-10-16T08:14:25.126371Z←[0m ←[32m INFO←[0m ←[2mbevy_render::renderer←[0m←[2m:←[0m AdapterInfo { name: "NVIDIA GeForce GTX 960", vendor: 4318, device: 5121, device_type: DiscreteGpu, backend: Vulkan }

How can I get rid of command line?

**Comment by ickk** [comment, marked as answer]
I think on windows all you need to do is add
```rust
#![windows_subsystem = "windows"]
```
to the top of `main.rs`

see: https://github.com/rust-lang/rust/pull/37501

**Reply by rth-8** [reply]
Thanks. That helped.
